terraform/helm/genesis/values.yaml

```diff
@@ -5,31 +5,32 @@ chain:
   era: 1
   # -- Aptos Chain ID
   chain_id: 4
-  # -- If specified, the root key for the testnet
+  # -- If true, genesis will create a resources account that can mint coins.
+  is_test: true
+  # -- If specified, the key for the minting capability in testnet
   root_key: "0x5243ca72b0766d9e9cbf2debf6153443b01a1e0e6d086c7ea206eaf6f8043956"
-  # -- Allow new validators
+  # -- Allow new validators to join after genesis
   allow_new_validators: false
-  # -- Length of each epoch in seconds. Defaults to 2 hours
-  epoch_duration_secs: 7200
-  is_test: true
-  # -- Minimum price per gas unit
-  min_price_per_gas_unit: 1
   # -- Minimum stake. Defaults to 1M APTOS coins with 8 decimals
   min_stake: 100000000000000
   # -- Mininum voting threshold. Defaults to 1M APTOS coins with 8 decimals
   min_voting_threshold: 100000000000000
-  # -- Maximum stake. Defaults to 1M APTOS coins with 8 decimals
+  # -- Required stake to be a proposer. 1M APTOS coins with 8 decimals
+  required_proposer_stake: 100000000000000
+  # -- Maximum stake. Defaults to 1B APTOS coins with 8 decimals
   max_stake: 100000000000000000
+  # -- Length of each epoch in seconds. Defaults to 2 hours
```

Comment:
nit" Why moved this down here? It was alphabetic :)

Reply:
It's easier to compare things with amount together, and things with duration together.